src/api/userService.ts

```diff
@@ -0,0 +1,25 @@
+import { defaultClient } from "./client";
+import type { LoginCredentials, RegisterCredentials } from "@/types/user";
+
+export const register = async (credentials: RegisterCredentials) =>
+  defaultClient.post("/register", {
+    user: {
+      email: credentials.email,
+      password: credentials.password,
+      date_of_birth: "1990-01-01",
```

Comment:
Date of birth is hard coded, not something that you want to do here :)

Reply:
the function is written in advance while working on user api service - registration to be implemented in next PR